### PR TITLE
[Docs] Standardization of the end of each documentation page - Get Started - Erik Review

### PR DESCRIPTION
**Pull request by rafael-rosa-knowcode**
GitHub Issue (If applicable): closes #

Related to https://github.com/unoplatform/uno/pull/14741

## PR Type

What kind of change does this PR introduce?

- Documentation content changes

## What is the current behavior?

In the Get Started section we do not have a pattern for finishing each page.

## What is the new behavior?

We have @eriklimakc  review updated.

image

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Review by eriklimakc — APPROVED** [review]
LGTM!

File: doc/articles/create-an-app-rider.md [hunk 1]
```diff
@@ -69,9 +69,7 @@ Select the Skia.WPF project, then Run.
 
 ## Next Steps
 
-Now that you're Created and Debug the App.
-
-Learn more about:
+Now that you've created and debugged the App, learn more about:
```
Comment (lukeblevins) [inline doc/articles/create-an-app-rider.md:72]:
```suggestion
Now that you've created and debugged the app, learn more about:
```

File: doc/articles/create-an-app-vs2022.md [hunk 2]
```diff
@@ -124,9 +124,7 @@ You're all set!
 
 ## Next Steps
 
-Now that you're Created and Debug the App.
-
-Learn more about:
+Now that you've created and debugged the App, learn more about:
```
Comment (lukeblevins) [inline doc/articles/create-an-app-vs2022.md:127]:
```suggestion
Now that you've created and debugged the app, learn more about:
```

File: doc/articles/create-an-app-vscode.md [hunk 3]
```diff
@@ -123,9 +123,7 @@ You can also find [additional VS Code topics](xref:Uno.vscode.additional), such
 
 ## Next Steps
 
-Now that you're Created and Debug the App.
-
-Learn more about:
+Now that you've created and debugged the App, learn more about:
```
Comment (lukeblevins) [inline doc/articles/create-an-app-vscode.md:126]:
```suggestion
Now that you've created and debugged the app, learn more about:
```

File: doc/articles/get-started-explore.md [hunk 4]
```diff
@@ -18,9 +18,7 @@ You may encounter issues while developing your app. Please see the [troubleshoot
 
 ## Next Steps
 
-Now that you have built your first application and Explored the Uno Platform
-
-Learn more about:
+Now that you have built your first application and explored the Uno Platform, learn more about:
```
Comment (lukeblevins) [inline doc/articles/get-started-explore.md:21]:
```suggestion
Now that you've built your first app and explored the Uno Platform, learn more about:
```

**Review by lukeblevins — COMMENTED** [review]
Some minor nits with capitalization, etc

**Review by lukeblevins — APPROVED** [review]

File: src/Uno.UWPSyncGenerator/Properties/launchSettings.json [hunk 5]
```diff
@@ -2,7 +2,7 @@
   "profiles": {
     "Uno.UWPSyncGenerator": {
       "commandName": "Project",
-      "commandLineArgs": "sync"
+      "commandLineArgs": "doc"
```
Comment (jeromelaban) [inline src/Uno.UWPSyncGenerator/Properties/launchSettings.json:5]:
```suggestion
      "commandLineArgs": "sync"
```

File: doc/articles/get-started-vscode.md [hunk 6]
```diff
@@ -74,8 +74,6 @@ You may need to follow additional directions, depending on your development envi
 
 ## Next Steps
 
-You're all set! Now that you're done with setting up your environment.
-
-Learn more about:
+You're all set! Now that you're done with setting up your environment, learn more about:
```
Comment (jeromelaban) [inline doc/articles/get-started-vscode.md:77]: Changes to the get started are not needed, they've been aligned in a separate PR.

File: src/Uno.UWPSyncGenerator/DocGenerator.cs [hunk 7]
```diff
@@ -140,6 +140,18 @@ void BuildMemberLists()
 									_sb.AppendParagraph($"In addition, {formattedViewName} has Uno-specific documentation {Hyperlink("here", customDocLink)}.");
 								}
 
+								var galleryLink = GetGalleryLink(viewName);
```
Comment (jeromelaban) [inline src/Uno.UWPSyncGenerator/DocGenerator.cs:143]: @rafael-rosa-knowcode  For some reason, it does not look that the link is generated properly: https://unodocsprstaging.z13.web.core.windows.net/pr-14774/articles/implemented/microsoft-ui-xaml-controls-combobox.html

ComboBox is in the list of gallery/playground links but it does not appear in the generated doc.

File: src/Uno.UWPSyncGenerator/Properties/launchSettings.json (hunk 5 above)
Comment (rafael-rosa-knowcode) [reply src/Uno.UWPSyncGenerator/Properties/launchSettings.json]: My mistake. It was in the wrong branch. The other is still in progress.
Reverted..

File: src/Uno.UWPSyncGenerator/DocGenerator.cs (hunk 7 above)
Comment (rafael-rosa-knowcode) [reply src/Uno.UWPSyncGenerator/DocGenerator.cs]: My mistake. It was in the wrong branch. The other is still in progress.
Reverted..

File: doc/articles/get-started-vscode.md (hunk 6 above)
Comment (rafael-rosa-knowcode) [reply doc/articles/get-started-vscode.md:77]: @jeromelaban 
In this case, can this PR be closed?
Wouldn’t standardizing ## Next Steps with text and topic be a good option?

**Comment by jeromelaban** [comment]
Let's close for now, we'll reevaluate when most changes are in.

**Pull request closed without merging**